src/main/java/seedu/address/logic/parser/cart/CartRemoveIngredientCommandParser.java

```diff
@@ -56,7 +56,7 @@ public CartRemoveIngredientCommand parse(String args) throws ParseException {
      * Returns true if none of the prefixes contains empty {@code Optional} values in the given
      * {@code ArgumentMultimap}.
      */
-    private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
+    static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```

Comment:
I used it in the tests

Reply:
On another note, I think we can maybe make this a public static method in a central class, maybe CookingPapaParser? It seems its being repeated in a few classes

Reply:
I think we don't have to test private methods? It doesn't seem like making these methods non-private for the sake of testing them is a good idea. The method would already be tested implicitly by testing the parsing, and the coverage in terms of lines executed would be the same. Also, testing all these methods seems quite inefficient in terms of uncovering new bugs that wouldn't also fail for the parsing methods.

Reply:
That makes sense. But regardless, it does seem like it is being repeated in 7 classes. Do you think we should move it to one location and make the 7 classes call that method?

Reply:
Yeah, that's a good idea. Perhaps we could put it in the `ArgumentMultimap` class directly.

Reply:
Doing so inevitably requires us to make it public/package-private though

Reply:
I think that's fine, because the `ArgumentMultimap` contains the prefix-argument mappings, and it seems to make sense to have a method where clients can check if certain prefixes are present, similar to `contains()` type of methods. We could even make it public, but at least we can cut down on the repeated code, which doesn't seem to fit very well in all the parser classes anyway.